### ISSUE TITLE
[Docs] New page for generic accessibility classes and attributes

### ISSUE THREAD

**Issue by lonix1** [twbs/bootstrap#37609]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed feature requests
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Proposal

There are quite a number of pages in the docs which discuss accessibility. 

It would be very helpful to have a new "accessibility" page which summarises all generic accessibility classes / aria attributes in one place. I'm not referring to accessibility specific to some component, but rather those bits that are generic.

For example, I always struggle to remember how to show visually and hide to screen readers, or vice versa (`aria-hidden="true"` and `.visually-hidden`). One of those has a dedicated page, and the other does not.

### Motivation and context

I refer to the docs regularly. And I always find it difficult to find the bit that I'm looking for - I can't remember if it's under the "text", "reboot", "visibility", "visually hidden", etc., pages.

**Comment by julien-deramond** [comment]
Are we talking here about improving https://getbootstrap.com/docs/5.2/getting-started/accessibility/ or creating something else?

**Comment by julien-deramond** [comment]
Yes some info about a11y is very generic (global classes, our philosophy, etc.), some is very specific to one component and will depend on the context of the use of one component. But I understand the point. I haven't checked everything but in the other pages, maybe we can link when possible to this main Accessibility page so that the users are used to go there, and try to gather in this main page most of the info. Let's try something :)
Thanks for the feedback btw!

**Comment by patrickhlauke** [comment]
just taking the example of, to paraphrase, `aria-hidden` not having a dedicated page...it's not really the job of the bootstrap docs to cover basic ARIA concepts in isolation
